index.html

```diff
@@ -0,0 +1,273 @@
+                    on any of the social media listed below if you would like to see more of 
+                    my work.
+                </p> 
+            </div>
+
+            <div class="connect">
+                <div id="title">LET’S CONNECT</div>
+                <ul class="social d-flex justify-start no-pad">
+                   <li>
+                        <a href="https://twitter.com/abiola_arobieke">
+                            <img src="./images/twitter-icon.png" alt="twitter-icon">
+                        <a/> 
+                  </li>
+                   <li>
+                        <a href="https://linkedin.com/in/abiola-arobieke">
+                            <img src="./images/linkedin-icon.png" alt="linkedin-icon">
+                        <a/> 
+                    </li>
+                   <li><a href="#"><img src="./images/medium-icon.png" alt="medium-icon"><a/> </li>
+                   <li> 
+                        <a href="https://github.com/abiola-arobieke">
+                            <img src="./images/github-icon.png" alt="github-icon">
+                        <a/> 
+                    </li>
+                   <li><a href="#"><img src="./images/hand-icon.png" alt="hand-icon"><a/> </li>
+                </ul>
+            </div>
+        </section>
+    </div>
+    <!-- end of header sections -->
+    
+    <!-- projects section -->
+    <div class="d-flex flex-column space-y-3 my-88 mx-24">
+        <section>
+            <div class="card d-flex flex-wrap">
+                <div>
+                    <img src="./images/project1.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Tonic</h1>
+                    <ul class="d-flex space-x-1 p-highlight">
+                        <li class="font-13">Facebook</li>
+                        <li>Full Stack</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        Experimental content creation feature 
+                        that allows users to add to an 
+                        existing story over the course of a day
+                         without spamming their friends.
+                    </p>
+                    <ul class="d-flex space-x-1 list-tags no-pad">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="btn-action" type="button">See project</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="card grid">
+                <div>
+                    <img src="./images/project2.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Multi-Post Stories</h1>
```

Comment:
- [ ] Based on best practices for semantics, you are expected to have just one `h1` element in a given project.
- It is recommended to use the heading hierarchy, where you use lower-level headings (h2, h3, etc.) after the main heading (h1), for several reasons:

- Accessibility: As I mentioned earlier, screen readers use the heading structure of a webpage to provide an overview of the content and navigate through it. By using a logical hierarchy of headings, you make it easier for screen readers to understand and convey the structure of the content to users.

- Semantic structure: HTML is a semantic markup language, meaning that it provides meaning and structure to content. Using a logical heading hierarchy provides semantic structure to your content, making it easier for search engines to understand and index your page.

- Visual hierarchy: Headings are also used to create a visual hierarchy on a page, with larger headings indicating more critical content. Using a logical hierarchy of headings, you can help users quickly scan and understand the content on your page.

- So, using lower-level headings after `h1` helps improve the accessibility, semantic structure, and visual hierarchy of your HTML content.